tests/keras/test_sequential_model.py

```diff
@@ -473,5 +485,52 @@ def test_nested_sequential_deferred_build():
     assert len(new_inner_model.weights) == 4
 
 
+@keras_test
+def test_sequential_1D():
```

Comment:
The tests added here seem like integration tests more than unit tests. Besides, it would help a lot with test readability if you don't defer to a heavily-parameterized utility function and instead explicitly test what you mean to test (what's actually getting tested is still unclear to me after reading the test).

E.g.

```python
(input_data, targets), _ = get_test_data(input_shape, ...)
assert model.compute_output_shape(input_shape) == expected_output_shape
output = model.predict(input_data)
assert output.shape == expected_output_shape
# etc.
```